pmc/challenges/default.json

```diff
@@ -143,6 +143,9 @@
         {
           "operation": {
             "operation-type": "force-merge",
+{%- if force_merge_max_num_segments is defined %}
```

Comment:
I think this added force merge max segments to `append-no-conflicts-index-only` challenge, but not the new one